CMSgov/onemac
# OY2 21167: SPA language correction for Cards & OY2 21194: "K" is missing for the Appendix K amendment title, package view.

Story: https://qmacbis.atlassian.net/browse/OY2-21167
Story: https://qmacbis.atlassian.net/browse/OY2-21194
Endpoint: See github-actions bot comment

### Details
Edit confusing language pattern on the triage card.
Add missing 'K' to the Appendix K RAI Response form details label.

### Changes
- Change the '&' to 'or' in the Submission Type choice card
- Added the 'K' to the appropriate place in the App 'K' RAI Response form

### Test Plan
1. Login as a submitting user
2. Navigate to Package Dashboard
3. Click New Submission button
4. Verify that text says or instead of &
5. Navigate to Package Dashboard - Waivers Tab
6. Locate an Appendix K in "RAI Issued" status
7. Click "Respond to RAI" action
8. Verify the Form Details label reads correctly.
